src/store.config.js

```diff
@@ -4,16 +4,61 @@ import reducers from './reducers/index';
 const storeConfigure = () => {
   const initialStore = {
     careers: [
-      { title: 'doctor',
-        tagline: 'saves lives',
-        description: 'this is the amazing doctor who is just really rich!',
-        image: 'doctor.svg'
+      { title: 'Doctor',
+        tagline: 'Promots and Improves Health',
+        description: 'A doctor is a professional who practises medicine, which is concerned with promoting, maintaining, or restoring health.',
+        image: 'doctor.svg',
+        degree: 'Medicine',
+        grade_bagrut: '11.10',
+        grade_psychometric: '760',
+        universities: 'Hebrew University, Technion, Ben-Gurion University, Tel-Aviv University',
+        salary_start: '9,000',
+        salary_ten_year: '35,000'
```

Comment:
Is this a yearly or monthly salary? And is it Shekels or dollars?

Reply:
It's monthly, in shekels. Want to move to Israel? :stuck_out_tongue_winking_eye:

Reply:
@veredrec I'm only staying if you are